cleverhans/loss.py

```diff
@@ -84,20 +109,38 @@ class CrossEntropy(Loss):
   :param model: Model instance, the model on which to apply the loss.
   :param smoothing: float, amount of label smoothing for cross-entropy.
   :param attack: function, given an input x, return an attacked x'.
+  :param pass_y: bool, if True pass y to the attack
   """
-  def __init__(self, model, smoothing=0., attack=None, **kwargs):
+  def __init__(self, model, smoothing=0., attack=None, pass_y=False,
+               adv_coeff=0.5, attack_params=None,
```

Comment:
can you add a docstring for adv_coeff and attack_params?

Reply:
thanks, added docstrings